torch/nn/modules/module.py

```diff
@@ -700,6 +700,18 @@ def bfloat16(self: T) -> T:
         """
         return self._apply(lambda t: t.bfloat16() if t.is_floating_point() else t)
 
+    def empty(self: T, device: Optional[Union[str, device]] = None) -> T:
+        r"""Moves the parameters and buffers to the specified device without copying storage.
+
+        Args:
+            device (:class:`torch.device`): The desired device of the parameters
+                and buffers in this module. If None, the current device is used. Default: None
```

Comment:
What does it mean to call this with `None` argument? Is it like a clone but generating random value? Is there any valid use case for that or do we want to make device a required argument?

Reply:
Yeah, it's like clone but without initializing any of the new weights. Not sure if there is a valid use case.

Reply:
Can't think of a valid use case either! The modules ends up with "random" (uninitialized) values for all params / buffers. 

I'm good with making `device` required.